packages/forms/lib/src/upload/components/upload-zone/upload-zone.component.html

```diff
@@ -22,7 +22,7 @@
 	<small class="m-upload__description"><ng-content select=".m-upload__description"></ng-content></small>
 </div>
 
-<button class="a-button m-upload__button" (click)="triggerFile()" *ngIf="uploader.options.type === 'button'">
-	<ng-content select=".m-upload__button"></ng-content>
-	<input type="file" #fileInput (change)="updateFiles()" multiple="multiple" class="m-upload__input">
+<button type="button" class="a-button m-upload__button" (click)="triggerFile()" *ngIf="uploader.options.type === 'button'">
+	<span [id]="ariaId"><ng-content select=".m-upload__button"></ng-content></span>
+	<input style="display:none" type="file" [attr.aria-labelledby]="ariaId" [id]="id" #fileInput (change)="updateFiles()" multiple="multiple">
```

Comment:
Why is `style="display:none"` not set in a CSS class?

Reply:
Fixed.